docs/workflows.rst

```diff
@@ -68,9 +68,9 @@ T1w/T2w preprocessing
                               reportlets_dir='.',
                               output_dir='.',
                               template='MNI152NLin2009cAsym',
-                              output_spaces=['T1w', 'fsnative',
-                                             'template', 'fsaverage5'],
-                              skull_strip_template='OASIS',
+                              fs_spaces=['T1w', 'fsnative',
```

Comment:
I assume `fs_spaces` means FreeSurfer? `T1w` and `template` don't seem appropriate entries...

Reply:
I renamed `output_spaces` in sMRIPrep because I still believe we need to handle that differently, but at this point it is just a renaming. I could roll it back in sMRIPrep, but honestly I don't think it is worth the pain.